packages/l/libcurl/xmake.lua

```diff
@@ -117,45 +120,45 @@ package("libcurl")
             io.replace("CMakeLists.txt", "list(APPEND CURL_LIBS OpenSSL::SSL OpenSSL::Crypto)", "list(APPEND CURL_LIBS OpenSSL::SSL OpenSSL::Crypto dl)", {plain = true})
             io.replace("CMakeLists.txt", "list(APPEND CURL_LIBS ${OPENSSL_LIBRARIES})", "list(APPEND CURL_LIBS ${OPENSSL_LIBRARIES} dl)", {plain = true})
         end
-        import("package.tools.cmake").install(package, configs, opt)
-    end)
-
-    on_install("android", function (package)
-        local configs = {"--disable-silent-rules",
-                         "--disable-dependency-tracking",
-                         "--without-hyper",
-                         "--without-libgsasl",
-                         "--without-librtmp",
-                         "--without-quiche",
-                         "--without-ngtcp2",
-                         "--without-nghttp3"}
-
-        local version = package:version()
-        if (package:is_plat("mingw") and version:ge("7.85")) then
-            package:add("syslinks", "Bcrypt")
-        end
-
-        table.insert(configs, "--enable-shared=" .. (package:config("shared") and "yes" or "no"))
-        table.insert(configs, "--enable-static=" .. (package:config("shared") and "no" or "yes"))
-        if package:debug() then
-            table.insert(configs, "--enable-debug")
-        end
-        if package:is_plat("macosx", "iphoneos") then
-            table.insert(configs, (package:version():ge("7.77") and "--with-secure-transport" or "--with-darwinssl"))
-        end
-        for _, name in ipairs({"openssl", "mbedtls", "zlib", "brotli", "zstd", "libssh2", "libidn2", "libpsl", "nghttp2"}) do
-            table.insert(configs, package:config(name) and "--with-" .. name or "--without-" .. name)
-        end
-        table.insert(configs, package:config("cares") and "--enable-ares" or "--disable-ares")
-        table.insert(configs, package:config("openldap") and "--enable-ldap" or "--disable-ldap")
-        if package:is_plat("macosx") then
-            local cares = package:dep("c-ares")
-            if cares and not cares:config("shared") then
-                -- we need fix missing `-lresolv` when checking c-ares
-                io.replace("./configure", "PKGCONFIG --libs-only-l libcares", "PKGCONFIG --libs-only-l --static libcares", {plain = true})
+        local function handledependency(conf, depname, includeconfig, libconfig)
+            if package:config(conf) then
+                local dep = package:dep(depname)
+                if dep and not dep:is_system() then
```

Comment:
why? fetch supports system libs

Reply:
I can remove it, but the idea is that cmake should be able to find it if it's a system library

Reply:
ok